app/src/main/java/org/oppia/android/app/walkthrough/WalkthroughViewModel.kt

```diff
@@ -1,11 +1,24 @@
 package org.oppia.android.app.walkthrough
 
 import androidx.databinding.ObservableField
+import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
 import org.oppia.android.app.viewmodel.ObservableViewModel
 import javax.inject.Inject
 
 /** [ViewModel] for [WalkthroughActivity]. */
 class WalkthroughViewModel @Inject constructor() : ObservableViewModel() {
   val currentProgress = ObservableField(0)
+  val isHeaderTextViewVisible = MutableLiveData<Boolean>(false)
```

Comment:
It seems like only one of the thing remains true while the other one will be false, then I think we should use only one variable to get it done.

Reply:
Done

Reply:
So, basically, this also means we need only one method in the interface.